crates/manifest/src/schema/common.rs

```diff
@@ -11,6 +11,13 @@ use super::json_schema;
 #[derive(Clone, Debug, Serialize, Deserialize, JsonSchema)]
 #[serde(deny_unknown_fields)]
```

Comment:
@radu-matei Note this means you won't be able to use this field with old versions of Spin.

Reply:
Yeah, saw that. It makes sense.